### DISCUSSION TITLE
Stay tuned for an update. We're Back!

### DISCUSSION THREAD

**Discussion by RyanMcEntire** [club-40/odin-pantry#40] [General]
It's completely natural and common for a project of this nature to have slowdowns and pauses. We're all volunteers working on our own path to a future in the industry. I want to thank @Philip-Clark for getting this project started and all of the infrastructure and plan laid out. Also of course all of the people who contributed so far. Getting things going is the hardest part.

Recently I've been itching for a project like this, so I checked in and found out it was on hiatus. I quickly volunteered to join as a lead, and I'll take the next couple days to get up to speed on the state of the project. I don't have the experience of working on a full stack app with a ruby backend, no less, but I know there are talented people in the community who have the experience to help with all of the things I have never done before. There have also been plenty of newcomers who might be interested in helping since the time this was last active and I'd like to give them a chance to contribute.

I'll make a post soon talking about the next steps. I intend to have a short list by then of plans and next steps, so we can start taking volunteers who will take ownership of the current blockers. It should be a lot of fun and I'm looking forward to it.

**Comment by RyanMcEntire** [comment]
@club-40/maintainers 

I'd like to do a quick roll call for maintainers to see if anybody is still interested in taking Odin Pantry to the next step. I understand that a lot can change in 7-8 months so there's no pressure to commit to the same role. I also understand some of you have already mentioned you have limited bandwidth or have to step down.

I'd like to see who has some time and is still interested. We can check if we need to onboard more people.

Based on my current understanding, we need volunteers who can assist with reviews for both front end and back end. We also need people who can take ownership of a module and break it down into actionable tasks. 

The stack has been decided, and I don't see a reason to change anything, so from here on out its mostly implementation, rather than group discussion on what choices to make across the app. It makes sense to me for individuals to take lead of features and make decisions on implementation. For a reminder on the stack: https://github.com/orgs/club-40/discussions/14#discussioncomment-5665730

First up for back end is the JWT-based authentication. If someone is confident in making that decision we can get rolling on that. 

Front end is getting going. Figma wireframe is done and storybook is getting planned out. We'll be building components soon.

**Reply by JoshDevHub** [reply]
@club-40/advisors @club-40/back-end-developers @club-40/front-end-developers ^^^

**Reply by Philip-Clark** [reply]
I'm available for small tasks like PR #33 
Depending on how things go with my job searching and learning, I might be able to dedicate more time in the new year.
